src/services/permissions.service.ts

```diff
@@ -0,0 +1,13 @@
+// TODO: Implement Permissions Service
+
+/**
+ * Used to indicate permissions levels for bot users.
+ * SUPER_ADMIN refers to administrators across all bot servers (i.e. bot maintainers).
+ */
+export enum PermissionsLevel {
+    EVERYONE,
+    USER,
```

Comment:
Personally - I think it makes sense to add `BOT` as the Bottom Level (since Bots should have almost no access to commands) and `VETERAN` as a Level between `USER` and `MOD`. I'd imagine a use case, for example:
- *other Bots* - `BOT`
- *literally everone* - `EVERYONE`
- *someone who has picked a role* - `USER`
- *Valued Contributor* - `VETERAN`
- ... (Rest should be obvious)

Reply:
Feel free to make the change. I wasn't going to include `BOT` only because we are currently ignoring bots for everything... But if you want to, add it, and what I'd suggest is set it to `BOT = -1`.